apps/experiments/models.py

```diff
@@ -830,14 +833,16 @@ def create_new_version(self, version_description: str | None = None, make_defaul
             self.versions.filter(is_default_version=True).update(
                 is_default_version=False, audit_action=AuditAction.AUDIT
             )
-
+        if copy_experiment:
+            new_version.name = new_version.name + "_copy"
+            new_version.version_number = 1
         new_version.save()
 
         self._copy_safety_layers_to_new_version(new_version)
         self._copy_routes_to_new_version(new_version)
```

Comment:
For now this should check if there are routes and raise an error since we aren't supporting copying experiments with child links.